### ISSUE TITLE
Bug affichage annotations numérisées à partir de document PDF

### ISSUE THREAD

**Issue by Balthazar45** [CNRS-LACITO/Pangloss_website#225]
Ce message pour signaler un bug sur le lecteur Eastling, lors de la lecture et de l’affichage d'annotations (images scannées pdf), réalisées à partir d'un document PDF.
l'exemple sélectionné est celui du corpus Cèmuhî ( https://doi.org/10.24397/pangloss-0005265).

Ci-dessous l'illustration du bug constaté. Lors de la consultation de la ressource, en affichage pleine largeur, le mode karaoké n'affiche pas tous les mots comme dans l'exemple ci-dessous :
![bug-blocs-annotations-pdf](https://github.com/CNRS-LACITO/Pangloss_website/assets/53433279/821ff968-c875-4083-9443-3aa1850332cb)
Autre bug, il y a une barre verticale, sous forme d'un curseur ou d'un bloc vide, qui s'affiche lors de la lecture sans aucune sélection de transcription des mots.
Ce curseur ou bloc correspond bien au choix de sélection de la transcription des mots. Mais il apparait touours, lorsque le choix n'est pas activé. exemple ci-dessous :
![bug-bloc-transcription-mots](https://github.com/CNRS-LACITO/Pangloss_website/assets/53433279/30ae1d45-4d54-496e-9e1c-f98988c136f9)Serait-il possible de résoudre ce dysfonctionnement et de réduire la résolution d'affichage des images PDF de ces annotations, pour un affichage sans être toujours en mode pleine largeur ?
Merci !

**Comment by m8nli9ht** [comment]
Ok vu. Note perso : ajouter l'instruction CSS "overflow:auto" sur le div conteneur et synchroniser le scroll avec le mot surligné

**Comment by sguillaume** [comment]
On en est où pour ce ticket ?

**Comment by alexis-michaud** [comment]
> Ok vu. Note perso : ajouter l'instruction CSS "overflow:auto" sur le div conteneur et synchroniser le scroll avec le mot surligné

Bonjour @m8nli9ht 
Comme le relève @sguillaume le bogue y est toujours. Quand prévois-tu de mettre en œuvre un plan d'action sur la base de la note perso que tu t'es faite il y a... 5 mois ? ;)

**Comment by m8nli9ht** [comment]
Je regarde ce week-end !

Le jeu. 2 mai 2024, 20:07, Alexis Michaud ***@***.***> a
écrit :

**Comment by EdouardSombie** [comment]
Salut Matthew,

J'espère que tu vas bien.
Je profite du fait que tu te penches sur le player pour te demander une chose :
Est-ce que le player pourrait déclencher un événement genre 'resourceError' et exposer une variable window.resourceErrorType qui aurait une valeur du type 500 ou 404...
Cette mécanique permettrait d'envoyer des notifications aux administrateurs du site en cas de ressource indisponible.

Excellent week end !
A bientôt

**Comment by m8nli9ht** [comment]
> Salut Matthew,
> 
> J'espère que tu vas bien. Je profite du fait que tu te penches sur le player pour te demander une chose : Est-ce que le player pourrait déclencher un événement genre 'resourceError' et exposer une variable window.resourceErrorType qui aurait une valeur du type 500 ou 404... Cette mécanique permettrait d'envoyer des notifications aux administrateurs du site en cas de ressource indisponible.
> 
> Excellent week end ! A bientôt

OK je vais regarder ça, peut-on mettre cette demande dans un autre ticket pour séparer ces deux sujets ?

**Comment by m8nli9ht** [comment]
J'ai corrigé et livré en DEV pour test : 

- adaptation du scroll selon la largeur de l'image
- suivi du scroll à la lecture
- camouflage du bloc Word quand pas de sélection d'options pour Word/Morphème

Quand vous aurez validé, je pousse en PROD

(pour la gestion d'erreur @EdouardSombie , on en parle dans un autre ticket !)

**Comment by Balthazar45** [comment]
Bonjour,

Je reviens sur ce ticket car de mon côté le bug d'affichage est toujours présent : 

![Capture d’écran 2024-10-30 à 09 37 26](https://github.com/user-attachments/assets/37221a7e-c25f-4911-b685-0447847e59d4)

Mais c'est peut-être en raison du livrable en DEV non encore poussé en PROD ?
Merci !

**Comment by Balthazar45** [comment]
Après vérification en mode DEV, voici le résultat : 

![Capture d’écran 2024-10-30 à 10 59 10](https://github.com/user-attachments/assets/e9e7567d-8d56-4a70-ac4e-c18450bf0ff9)

Le défaut d'affichage se porte à présent sur la hauteur lors de l'affichage en mode karaoké.
Merci !

**Comment by alexis-michaud** [comment]
un espoir pour 2025 ? @sguillaume @EdouardSombie

**Comment by sguillaume** [comment]
Oui, ça devrait être réglé en 2025 lors du re-développement du lecteur.

**Comment by Balthazar45** [comment]
Affichage, annotations numérisées à partir de documents PDF, résolu avec le développement du lecteur : 

<img width="1287" height="959" alt="Image" src="https://github.com/user-attachments/assets/14279a40-0858-48d3-a0b9-233e0bc00543" />.   



En revanche, une reprise et des corrections des annotations sont à réaliser pour cette ressource.
